lms/djangoapps/grades/new/course_grade.py

```diff
@@ -24,8 +27,40 @@ class CourseGrade(object):
     def __init__(self, student, course, course_structure):
         self.student = student
         self.course = course
+        self.course_version = getattr(course, 'course_version', None)
+        self.course_edited_timestamp = getattr(course, 'subtree_edited_on', None)
         self.course_structure = course_structure
-        self.chapter_grades = []
+        self._percent = None
+        self._letter_grade = None
+        self.subsection_grade_factory = SubsectionGradeFactory(self.student, self.course, self.course_structure)
+
+    @classmethod
+    def load_persisted_grade(cls, user, course, course_structure):
```

Comment:
Nit: Can you move this method down in the class definition?  It's not really a public method that we expect external callers to use.